.gitattributes

```diff
@@ -1,6 +1,5 @@
 # `git apply` and friends don't understand CRLF, even on windows. Force those
 # files to be checked out with LF endings even if core.autocrlf is true.
-*.patch text eol=lf
```

Comment:
We shouldn't remove this imo - it is more likely to cause further issues in future patch changes.

Reply:
> We shouldn't remove this imo - it is more likely to cause further issues in future patch changes.

The fix to crbug.com/383772517 patches the sources of libzma to move it to a secure version. The libzma sources consist of a mixture of LF and CRLF files. Forcing the patch file to be LF makes it fail to apply to the CRLF files. Is there another way to achieve this?

Reply:
@ppontes having patches with mixed line endings will break the patch apply process.  I think typically how we have handled this in the past is to convert any CRLF files patched to LF.  It will make the patch file bigger but unfortunately I don't think there's a better way to handle this.